### ISSUE TITLE
Consider a `bind` export for qualified-do

### ISSUE THREAD

**Issue by maddie927** [garyb/purescript-indexed-monad#15]
Adding an extra export of `ibind` as `bind` would allow `ibind` to be used with the qualified-do syntax:

```purs
import Control.Monad.Indexed as Ix

foo = Ix.do
  ...
  ...
```

If this seems fine I'll make a PR

**Comment by garyb** [comment]
Yeah, definitely 👍 

It's a shame I didn't just call it `bind` in the class. Maybe we should make a `Control.Monad.Indexed.Do` for any relevant aliases? (Could have `discard` too).

**Comment by JordanMartinez** [comment]
This was closed by #19 (but I can't close this issue)
